### PR TITLE
OCPBUGS-11382: Update Dockerfile to use latest builder images

### PR DESCRIPTION
**Pull request by TheRealJon**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,11 +21,11 @@ ARG YARN_VERSION=v1.22.19
 
 # bootstrap yarn so we can install and run the other tools.
 RUN CACHED_YARN=./artifacts/yarn-${YARN_VERSION}.tar.gz; \
-    if [ -f ${CACHED_YARN} ]; then \
-      npm install ${CACHED_YARN}; \
-    else \
-      npm install https://github.com/yarnpkg/yarn/releases/download/${YARN_VERSION}/yarn-${YARN_VERSION}.tar.gz; \
-    fi
+  if [ -f ${CACHED_YARN} ]; then \
```
Comment (spadgett) [inline Dockerfile:24]: Did you mean to reformat the Dockerfile?

**Review by spadgett — COMMENTED** [review]
Thanks @TheRealJon

File: Dockerfile (hunk 1 above)
Comment (TheRealJon) [reply Dockerfile:24]: Not really, my IDE just autoformats on save. I can revert if needed.

Comment (spadgett) [reply Dockerfile:24]: I find the previous indentation easier to read if you can temporarily disable that

**Comment by TheRealJon** [comment]
@spadgett I've updated this PR to revert the formatting changes

**Comment by TheRealJon** [comment]
@spadgett I've updated this PR to revert the formatting changes

**Review by spadgett — APPROVED** [review]
/lgtm

**Review by jcaianirh — APPROVED** [review]
/lgtm

**Pull request merged**